world-if.css

```diff
@@ -0,0 +1,196 @@
+/* HEADER */
+.world-if-ArticleTemplate--imagecontainer-inner {
+  position: relative;
+}
+
+.world-if-ArticleTemplate--header {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  padding: 0;
+  background-color: rgba(0,0,0,.6);
+}
+
+.world-if-ArticleTemplate--section-section {
+  display: inline-block;
+  font-family: Neutra2Display-Medium;
```

Comment:
Centralise repeated values via variables will be a NTH:
:root {
  --displayMediumFont: Neutra2Display-Medium;
}

then...

font-family: var(--displayMediumFont);